distbench_engine.cc

```diff
@@ -783,13 +824,49 @@ void DistBenchEngine::ActionListState::CancelActivities() {
   int size = action_list->proto.action_names_size();
```

Comment:
We probably should just use the expression instead of the generic 'size' variable In these cases.